plantcv/geospatial/points2roi.py

```diff
@@ -0,0 +1,55 @@
+# Transform georeferenced GeoJSON/shapefile points into python coordinates
+import geopandas
+from plantcv.geospatial.transform_polygons import transform_polygons
+from plantcv.plantcv import Objects
+
+
+def points2roi_circle(img, geojson, radius):
+    """Takes a points-type shapefile/GeoJSON and transforms circular ROIs,
+    saves these out to a new geoJSON file and creates ROI Objects instances
+    Inputs:
+    img:        A spectral object from read_geotif.
+    geojson:    Path to the shape file containing the points.
+    radius:     Radius of circular ROIs to get created,
+                in units matching the coordinate system (CRS) of the image
+                e.g. meters
```

Comment:
For the radius parameter, the key requirement is that the TIF and shapefile coordinate system should use meter-based coordinates. Our current algorithm uses CRS EPSG:32615, which has meter-based coordinates, but any coordinate system with meter-based units would be compatible.